docs/PythonAPI.md

```diff
@@ -0,0 +1,867 @@
+# API Reference
+
+Information on the classes in Acquire-python along with the attributes and methods associated with them.
+
+## Class `AvailableData`
+
+The `AvailableData` class represents the collection of frames that have been captured since the last call to runtime.get_available_data(). `AvailableData` objects should be set to have a short lifetime, since these objects reserve space on the video queue and will eventually block camera acquisition to ensure no data is overwritten before it can be processed.
+
+```python
+class AvailableData:
+    def frames(self) -> Iterator[VideoFrame]:
+        """Returns an iterator over the video frames in the available data."""
+    
+    def get_frame_count(self) -> int:
+        """Returns the total number of video frames in the available data."""
+    
+    def __iter__(self) -> Iterator[VideoFrame]:
+        """Returns an iterator over the video frames in the available data."""
+```
+- Call `get_frame_count()` to query the number of frames in an `AvailableData` object. 
+
+- The `frames` method provides an iterator over these frames.
```

Comment:
Maybe add a note here that the `AvailableData` object itself can be iterated over because of the `__iter__` method.